app/components/SearchBox.module.css

```diff
@@ -0,0 +1,82 @@
+.form {
+  width: 60%;
+  min-width: 329px;
+  /* this can be moved up and down pretty easily, im not satisfied with where its at */
+  /* margin-bottom: 60px; */
+}
+
+.searchBox {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 50px;
+  background-color: #f1f1f1;
+  border-radius: 25px;
+  padding: 0 15px;
+  outline: #2b2b2b solid;
+  min-width: 329px;
+}
+
+.searchIcon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #333333;
+  border-radius: 0 25px 25px 0;
+  width: 80px;
+  height: 52px;
+  margin-right: -16px;
+  outline: #2b2b2b solid;
+  outline-offset: -1px;
+}
+
+.searchIcon i,
+.locationIcon i {
+  font-size: 20px;
+  color: #ffffff;
+}
+
+.locationIcon {
+  display: flex;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  margin-left: 5px;
+}
+
+.locationIcon i {
+  font-size: 24px;
+  color: darkred;
+}
+
+.searchBox input {
+  flex-grow: 1;
+  height: 100%;
+  border: none;
+  background-color: transparent;
+  outline: none;
```

Comment:
not a good practice here - elements that can be interacted with should have outlines.  let's fix this in another PR along with the button